gradle-plugins/plugin-base/src/main/java/io/spine/gradle/SpinePlugin.java

```diff
@@ -158,6 +168,53 @@ public Builder insertAfterTask(TaskName target) {
             }
 
             /**
+             * Specify tasks which will precede the new one.
+             *
+             * <p>Unlike {@link #insertAfterTask insertAfterTask()}, this method will depend
+             * the new task on <b>every</b> task with such name in the project (i.e. the tasks of
+             * the root project and all the subprojects).
+             *
+             * <p>If a certain project does not have a task with the specified name, no action is
+             * performed for that project.
+             *
+             * <p>This method does not guarantee that the task will be depended on either task.
```

Comment:
This isn't clear. That's the first time "to depend" is used, maybe it's a root issue.